docs/reference/commandline/dockerd.md

```diff
@@ -740,6 +740,19 @@ to add multiple lower directory support for OverlayFS. This option should
 only be used after verifying this support exists in the kernel. Applying
 this option on a kernel without this support will cause failures on mount.
 
+##### `overlay2.size`
```

Comment:
Ohhh, the CLI reference was moved to the https://github.com/docker/cli repository (sorry! it was a long time in the making, and we finally were able to move them). Although `dockerd` itself is in this repository, this document was moved there as well; https://github.com/docker/cli/blob/master/docs/reference/commandline/dockerd.md

Could you open a separate pull-request in that repository, and remove the change here?